pipelines/hf_radar/pipeline.py

```diff
@@ -43,11 +44,11 @@ def hook_plot_dataset(self, dataset: xr.Dataset):
         plt.style.use("default")  # clear any styles that were set before
         plt.style.use("shared/styling.mplstyle")
 
-        with self.storage.uploadable_dir(datastream) as tmp_dir:
+        with self.storage.uploadable_dir() as tmp_dir:
```

Comment:
This is still using the old way of writing plots. It should be like this:

```python
with plt.style.context("shared/styling.mplstyle"):
    # regular plotting code
    fig, ax = plt.subplots(1, 2, figsize=(12, 10))
    ...

    plot_file = self.get_ancillary_filepath("surface_velocity")
    fig.savefig(plot_file)
    plt.close(fig)
````


For reference you can look at the [flux sonic summary](https://github.com/a2edap/ingest-wfip3/blob/main/pipelines/sonic/flux_sonic_summary_vap/pipeline.py#L27) pipeline.